test/index.spec.js

```diff
@@ -1,7 +1,7 @@
 var should = require('chai').should();
 var expect = require('chai').expect;
 
-var VT = require('../dist/index.js');;
+var VT = require('../src');;
```

Comment:
Run on source, as you can do `--compilers js:babel-core/register` while running tests.